src/components/AddPeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -360,7 +362,7 @@ export default class PeopleAndRoles extends Mixins(EntityFilterMixin) {
       // remove the Completing Party role
       completingParty.roles = completingParty.roles.filter(role => role.roleType !== RoleTypes.COMPLETING_PARTY)
       // remove email address that we got from user profile
-      completingParty.officer.email = null
+      delete completingParty.officer.email // schema doesn't accept null or empty string
```

Comment:
Interesting, haven't used this much.

Reply:
@andrepestana-aot Another way to do this is to remove this property when the IA is saved (if it's falsey). What do you think?

Reply:
> @andrepestana-aot Another way to do this is to remove this property when the IA is saved (if it's falsey). What do you think?

Yes, either way work. I prefer this way.

Reply:
> Yes, either way work. I prefer this way.

Ok by me!